### PR TITLE
5.2

### PR DESCRIPTION
**Pull request by 2BriTo**
Update ElasticSearch-client to ElasticSearch 5 version

### CODE REVIEW

File: src/Clause.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: cheaphasz
+ * Date: 18/11/17
+ * Time: 17:04
+ */
```
Comment (skwi) [inline src/Clause.php]: These comments can be removed since they add no value.

File: src/Filter/NestedFilter.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    public function addClause(Clause $clause)
+    {
+        $this->clauses[] = $clause;
+    }
+
+    public function formatForQuery(): array
+    {
+        ;
```
Comment (skwi) [inline src/Filter/NestedFilter.php]: Oops ^_^

File: src/Index.php [hunk 3]
```diff
@@ -88,14 +88,18 @@ public function delete(array $params)
     public function search(array $searchParams, ResultTransformer $resultTransformer = null)
     {
         $searchParams['index'] = $this->name;
-
         $searchResult = $this->client->search($searchParams);
+        $limit = isset($searchParams['body']['size']) ? $searchParams['body']['size'] : null;
```
Comment (skwi) [inline src/Index.php]: The library requires the usage of PHP 7+ so this line can be replaced by 
```php
$limit = $searchParams['body']['size'] ?? null;
```

**Comment by CheapHasz** [comment]
As it is only ports of 2.3 changes, and every test is passing, LGTM

File: src/Query/Query.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: cheaphasz
+ * Date: 18/11/17
+ * Time: 17:04
+ */
```
Comment (skwi) [inline src/Query/Query.php]: Same as previous, these comments can be removed

**Review by skwi — COMMENTED** [review]
The overall code is OK, but there are some codestyle issues. I'll implement a stylechecker in a near future so it's ok if you leave it.

**Pull request merged**